scripts/src/main/resources/scripts/functions

```diff
@@ -1824,53 +1830,142 @@ function doGetSoftwareVersion() {
   fi
 }
 
+# $1: software
+function doGetSoftwareEdition() {
+  local software="${1}"
+  local edition
+  echo "${software}" | grep '-' >/dev/null
+  ERG="$?"
+  if [ "${ERG}" = "0" ]
+  then
+    echo "${software}"
```

Comment:
I do not really get this? If the software contains a hyphen, it can not have an edition?
So if this is the case for tools like `kotlin-native` you should instead simply use `${software/-/_}` for creating the variable name for the edition.

Reply:
With the agreement that a "-" in the command name leads to a "_" in the variables, the code can be simplified accordingly and is now valid without restrictions.